### PR TITLE
10.3.0 consolidated envs

### PR DESCRIPTION
**Pull request by jasperRob**
This will resolve some remaining cache issues.

### CODE REVIEW

**Review by varunshastry — APPROVED** [review]

File: kogitoq/kogito-common/src/main/java/life/genny/kogito/common/core/Dispatch.java [hunk 1]
```diff
@@ -393,9 +395,18 @@ public void handleDropdownAttributes(Ask ask, String parentCode, BaseEntity targ
 		// check for dropdown attribute
 		if (ask.getQuestion().getAttribute().getCode().startsWith(Prefix.LNK_)) {
 
-			// get list of value codes
-			List<String> codes = beUtils.getBaseEntityCodeArrayFromLinkAttribute(target,
-					ask.getQuestion().getAttributeCode());
+			List<String> codes = null;
```
Comment (Expressionless) [inline kogitoq/kogito-common/src/main/java/life/genny/kogito/common/core/Dispatch.java]: set to new ArrayList<>() here. SONARLint compliant + on line 411 means we only have to check if codes.isEmpty (code cleanliness)

File: dropkick/src/main/java/life/genny/dropkick/live/data/InternalConsumer.java [hunk 2]
```diff
@@ -214,18 +214,14 @@ public void getEvent(String event) {
 		log.info("DROPDOWN :Loaded " + msg.getItems().size() + " baseentitys");
 
 		for (BaseEntity item : msg.getItems()) {
-			String name = null;
+			String name = item.getName();
```
Comment (Expressionless) [inline dropkick/src/main/java/life/genny/dropkick/live/data/InternalConsumer.java:217]: if item.getName() is null would it not make sense to check PRI_NAME? (Compatibility)
Further we could make a more descriptive log then logging the same thing but as a warning to suggest that the base entity name is null.

Comment (jasperRob) [reply dropkick/src/main/java/life/genny/dropkick/live/data/InternalConsumer.java:217]: I would rather not use PRI_NAME. It just adds confusion.

File: kogitoq/kogito-common/src/main/java/life/genny/kogito/common/core/Dispatch.java (hunk 1 above)
Comment (jasperRob) [reply kogitoq/kogito-common/src/main/java/life/genny/kogito/common/core/Dispatch.java]: Done

**Review by Expressionless — APPROVED** [review]
lgtm

**Pull request merged**